source/Nevermore/UpgradeScripts/Script0003-RelatedDocumentType.sql

```diff
@@ -0,0 +1 @@
+﻿CREATE TYPE dbo.RelatedDocumentTableValuedParameter AS TABLE ([Id] NVARCHAR(400), [Table] NVARCHAR(400), [RelatedDocumentId] NVARCHAR(400), [RelatedDocumentTable] NVARCHAR(400))
```

Comment:
The type for the table valued parameter needs to be created upfront. We have prior art here with [ParameterList](https://github.com/OctopusDeploy/Nevermore/blob/d50d9c0dd50cadd468cad1dfcfddb271586b2c14/source/Nevermore/UpgradeScripts/Script0002-ParameterList.sql#L2-L1)